docs/en/code/index.md

```diff
@@ -0,0 +1,50 @@
+#Extending
+
+To change how the front-end javascript displays the map the custom script with the unique id `locator_map_init_script`.
+This example shows how to reference custom templates and a custom data location. It is recommended to copy the custom script from the [Locator Controller](../../../code/pages/Locator.php#L273-L298) and modify it.
+
+```php
+class Locator_ControllerExtension extends DataExtension

```

Comment:
If this is applied to the `Locator_Controller` it should extend `Extension` rather than `DataExtension`.